src/include/daos_fs_sys.h

```diff
@@ -27,15 +25,15 @@ extern "C" {
 #include <daos.h>
 #include <daos_fs.h>
 
-#define DFS_SYS_NO_CACHE 1
-#define DFS_SYS_NO_LOCK 2
+/** Mount flags for dfs_sys_mount. By default, mount with caching and locking turned on. */
```

Comment:
This comment is being picked up by doxygen, and applied only to DFS_SYS_NO_CACHE, so I think you want to remove the second * here and add a < on the line below.

Reply:
Ah, Thanks!

Reply:
Looking through the code I see other instances of `/**<` so that's that I went with

Reply:
So I think now you've got three doxygen comments, one on L28 that doesn't apply to any code, then two on L29 and L30 that apply to the defines on those lines.  It's certainly enough to make doxygen happy but like all these things it'll be worth reading the generated html at some point.

Reply:
Where can I find the generated html?

Reply:
I don't think we save them in CI anywhere, but run "doxygen Doxyfile" and you should get a html directory to poke around in.

Reply:
It seems the comment above and beside DFS_SYS_NO_CACHE are concatenated into a single doc comment.
![image](https://user-images.githubusercontent.com/13281535/130265316-5a6f0c05-ea5b-456d-8f0d-270c6c92183b.png)

If it's okay with you, I'd like to NOT push another commit just for this.